### DISCUSSION TITLE
Why MAA cannot start annihilation ?

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#7271] [Q&A]
Hi,why MAA cannot start annihilation ? Can MAA do incomplete annihilation ?

**Comment by status102** [comment]
剿灭模式旨在完成每周剿灭周限，而不是用于通关剿灭

**Reply by zzyyyl** [reply]
btw, 你要是想通关剿灭可以到[作业站](https://prts.plus)找找作业
